src/librustc/hir/lowering.rs

```diff
@@ -4008,7 +4008,7 @@ impl<'a> LoweringContext<'a> {
         let attrs = self.lower_attrs(&i.attrs);
         if let ItemKind::MacroDef(ref def) = i.node {
             if !def.legacy || attr::contains_name(&i.attrs, sym::macro_export) ||
-                              attr::contains_name(&i.attrs, sym::rustc_doc_only_macro) {
+                              attr::contains_name(&i.attrs, sym::rustc_builtin_macro) {
```

Comment:
Is this needed anymore? The `macro_rules!` -> `macro` change would mean `!def.legacy` is now `true`.

Reply:
No, this is no longer needed.